code/clue.js

```diff
@@ -1,26 +1,131 @@
-// CREATE OBJECTS FOR ALL THE SUSPECTS, SOMETHING LIKE THIS:
-
+// OBJECTS FOR ALL THE SUSPECTS
 const mrGreen = {
   firstName: "Jacob",
   lastName: "Green",
-  color: "green",
+  color: "rgba(48, 75, 59, 0.8)",
   description: "He has a lot of connections",
-  age: 45,
-  image: "assets/green.png",
+  age: "Age: 45",
+  image: "assets/green.jpg",
   occupation: "Entrepreneur"
 }
 
-// CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS THE THE WEAPONS IF YOU LIKE.
+const profPlum = {
+  firstName: "Victor",
+  lastName: "Plum",
+  color: "rgba(69, 33, 68, 0.8)",
+  description: "He is widely seen as the intellectual",
+  age: "Age: 36",
+  image: "assets/plum.jpg",
+  occupation: "Professor"
+}
 
-const rope = {
-  name: "rope",
-  weight: 10
+const missScarlet = {
+  firstName: "Cassandra",
+  lastName: "Scarlet",
+  color: "rgba(153, 64, 62, 0.8)",
+  description: "She is typically portrayed as young, cunning, and highly attractive",
+  age: "Age: 25",
+  image: "assets/scarlet.jpg",
+  occupation: "Actress"
 }
 
-// THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
+const mrsPeacock = {
+  firstName: "Eleanor",
+  lastName: "Peacock",
+  color: "rgba(16, 29, 66, 0.8)",
+  description: "She is an elderly yet still attractive woman who maintains her dignity in almost all cases",
+  age: "Age: 60",
+  image: "assets/peacock.jpg",
+  occupation: "Socialite"
+}
 
-// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS.
+const mrMustard = {
+  firstName: "Jack",
+  lastName: "Mustard",
+  color: "rgba(228, 179, 99, 0.8)",
+  description: "He is usually a military man both dignified and dangerous",
+  age: "Age: 55",
+  image: "assets/mustard.jpg",
+  occupation: "Colonel"
+}
+
+const mrsWhite = {
+  firstName: "Blanche",
+  lastName: "White",
+  color: "rgba(232, 233, 235, 0.8)",
+  description: "She is a frazzled servant",
+  age: "Age: 65",
+  image: "assets/white.jpg",
+  occupation: "Housekeeper"
+}
+
+// OBJECTS FOR ALL THE WEAPONS
+const rope = {
+  name: "Rope",
+  weight: "Weight: 2kg",
```

Comment:
The same goes here as before on the `age`. It seems like you have decided that the weight always is in `kg`, so you can use just numbers here as well. So this could be `weight: 2`